numpy/core/src/multiarray/datetime_strings.c

```diff
@@ -1075,7 +924,7 @@ lossless_unit_from_datetimestruct(npy_datetimestruct *dts)
  */
 NPY_NO_EXPORT int
 make_iso_8601_datetime(npy_datetimestruct *dts, char *outstr, int outlen,
-                    int local, NPY_DATETIMEUNIT base, int tzoffset,
+                    int local, int utc, NPY_DATETIMEUNIT base, int tzoffset,
```

Comment:
Added a note.

I'm not really happy with defining a new variable `utc` here -- I'd rather use a string or some sort of enum type as the function parameter -- but this was the easiest thing to do (given my poor knowledge of C).